pkg/odo/cli/component/create.go

```diff
@@ -347,15 +352,13 @@ func (co *CreateOptions) Complete(name string, cmd *cobra.Command, args []string
 
 			// devfile.yaml is not present, user has to specify the component type
 			// Component type: We provide supported devfile component list then let you choose
-			if !util.CheckPathExists(DevfilePath) {
```

Comment:
this check is not needed because we are already checking it above.

Reply:
This is needed because:
- If devfile is present we **only** let user specify component name and namespace
- If devfile is not present, we let user specify component type, name and namespace
If we use the above checking, if devfile is present it will error our and return instead of falling down to interactive mode